### DISCUSSION TITLE
Eigenvalue Analysis

### DISCUSSION THREAD

**Discussion by wigoe** [optimatika/ojAlgo#490] [General, ojAlgo and related code]
Any good examples of using ojAlgo Matrix to get eigenvalues and eigenvectors?

**Comment by apete** [comment]
All example code can be found at: https://www.ojalgo.org/code-examples/

Go to that page and search for "eigen" and you'll find something. Other than that you can look at the junit test code for the various matrix decompositions.

**Reply by wigoe** [reply]
Thanks... I did get as far as 

Eigenvalue.PRIMITIVE.makeGeneralised(M)

Bill
